esp-hal/src/lib.rs

```diff
@@ -194,6 +194,9 @@ pub mod debugger;
 pub mod sync;
 pub mod time;
 
+#[cfg(tsens)]
+pub mod tsens;
```

Comment:
This needs to be in the unstable module list.

Reply:
Done 👍🏼